shared/chat/conversation/list.desktop.js

```diff
@@ -142,6 +144,27 @@ class ConversationList extends Component<void, Props, State> {
     this._onScrollSettled()
   }, 100)
 
+  showPopup (message: Message, event: any) {
+    if (message.type !== 'Text') return
+    const clientRect = event.target.getBoundingClientRect()
```

Comment:
no i mean just use the browsers dom flow. When they click the '...' just add the popup as a child of the '...' button. If you make that thing position relative you can have the position absolute popup be a child of it. i think you can do this w/o having to have the global popover layer